src/prompts/list.ts

```diff
@@ -10,7 +10,12 @@ export default class ListPrompt extends Prompt {
 
 	public render() {
 		const choices = this._question.choices.reduce((result, choice) => {
-			result[choice.name] = choice.value;
+			// choices can be string array or array of { name, value }
+			if (choice.name) {
```

Comment:
Maybe we should only fix the scoped generator issue in this PR and keep this change for a new PR that resolves #50 entirely?